From 5786d6ef42d7329b8d9000761db0a49a2b8cd6a0 Mon Sep 17 00:00:00 2001
From: Semih Yavuz <semih.yavuz@qt.io>
Date: Mon, 13 Oct 2025 13:15:20 +0200
Subject: [PATCH 178/239] style: remove indentation in namespace

Task-number: QTBUG-140645
Change-Id: I002346ee4623d87dcf06484000a826958bbf2a64
Reviewed-by: Fabian Kosmale <fabian.kosmale@qt.io>
(cherry picked from commit fbf320ee438cd9cd1cbccecfa7a010e640167d78)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/qmlls/qqmlsemantictokens_p.h | 30 +++++++++++++++---------------
 1 file changed, 15 insertions(+), 15 deletions(-)

diff --git a/src/qmlls/qqmlsemantictokens_p.h b/src/qmlls/qqmlsemantictokens_p.h
index 0c4d3790cc..b8edc9d3b6 100644
--- a/src/qmlls/qqmlsemantictokens_p.h
+++ b/src/qmlls/qqmlsemantictokens_p.h
@@ -150,21 +150,21 @@ struct HighlightsRange
 
 namespace Utils
 {
-    QList<int> encodeSemanticTokens(const HighlightsContainer &highlights, HighlightingMode mode = HighlightingMode::Default);
-    QList<QQmlJS::SourceLocation>
-    sourceLocationsFromMultiLineToken(QStringView code,
-                                      const QQmlJS::SourceLocation &tokenLocation);
-    void addModifier(QLspSpecification::SemanticTokenModifiers modifier, int *baseModifier);
-    bool rangeOverlapsWithSourceLocation(const QQmlJS::SourceLocation &loc, const HighlightsRange &r);
-    QList<QLspSpecification::SemanticTokensEdit> computeDiff(const QList<int> &, const QList<int> &);
-    void updateResultID(QByteArray &resultID);
-    QList<int> collectTokens(const QQmlJS::Dom::DomItem &item,
-                             const std::optional<HighlightsRange> &range,
-                             HighlightingMode mode = HighlightingMode::Default);
-    HighlightsContainer visitTokens(const QQmlJS::Dom::DomItem &item,
-                                 const std::optional<HighlightsRange> &range);
-    void addHighlight(HighlightsContainer &out, const QQmlJS::SourceLocation &loc, QmlHighlightKind,
-                      QmlHighlightModifiers = QmlHighlightModifier::None);
+QList<int> encodeSemanticTokens(const HighlightsContainer &highlights, HighlightingMode mode = HighlightingMode::Default);
+QList<QQmlJS::SourceLocation>
+sourceLocationsFromMultiLineToken(QStringView code,
+                                    const QQmlJS::SourceLocation &tokenLocation);
+void addModifier(QLspSpecification::SemanticTokenModifiers modifier, int *baseModifier);
+bool rangeOverlapsWithSourceLocation(const QQmlJS::SourceLocation &loc, const HighlightsRange &r);
+QList<QLspSpecification::SemanticTokensEdit> computeDiff(const QList<int> &, const QList<int> &);
+void updateResultID(QByteArray &resultID);
+QList<int> collectTokens(const QQmlJS::Dom::DomItem &item,
+                            const std::optional<HighlightsRange> &range,
+                            HighlightingMode mode = HighlightingMode::Default);
+HighlightsContainer visitTokens(const QQmlJS::Dom::DomItem &item,
+                                const std::optional<HighlightsRange> &range);
+void addHighlight(HighlightsContainer &out, const QQmlJS::SourceLocation &loc, QmlHighlightKind,
+                    QmlHighlightModifiers = QmlHighlightModifier::None);
 } // namespace Utils
 
 class HighlightingVisitor
-- 
2.51.2

